research-activities/card-sort.md

```diff
@@ -32,7 +32,7 @@ of the existing ones.
 
 ## Instructions
 
-1. Prepare category cards for each participant. *Write categories on cards for a
+1. Prepare category cards for each participant. \*Write categories on cards for a
```

Comment:
Given the way the "Things to remember" section is presented, I think the stray asterisks in the "Instructions" were originally meant to be part of emphasising the first sentence in the paragraph.

E.g. "1. *Prepare category cards for each participant.*" followed by regular text.

OR 

"1. **Prepare category cards for each participant.**" followed by regular text.

In general, if we find we have to escape characters in text meant to be authored by non-developers, I think we should choose the least technically complicated option.